wandb/sdk/internal/file_stream.py

```diff
@@ -236,6 +241,7 @@ def _thread_body(self):
         posted_anything_time = time.time()
         ready_chunks = []
         finished = None
+        last_log = 0
```

Comment:
I think this is a remnant of a change I was making that I discarded

Reply:
@KyleGoyette , removed, thanks